src/biokbase/narrative/tests/narrative_mock/mockcomm.py

```diff
@@ -15,15 +15,20 @@ def last_message(self):
             return None
         return self.messages[-1]
 
+    def pop_message(self):
+        if len(self.messages) == 0:
+            return None
+        return self.messages.pop()
+
     def peek_message(self, i):
         return self.messages[i]
 
     def on_msg(self, *args, **kwargs):
         """Mock the msg router"""
 
-    def send(self, data=None, content=None):
+    def send(self, message=None):
         """Mock sending a msg"""
-        self.messages.append({"data": data, "content": content})
```

Comment:
this was creating a strange data structure with an extra "content" key, so I changed it so it just saves the message without altering the structure.